packages/element/src/renderElement.ts

```diff
@@ -602,6 +602,9 @@ const generateElementWithCanvas = (
   }
   return prevElementWithCanvas;
 };
+function isEmojiOnly(text: string) {
```

Comment:
Why the separate function for a non-reusable test? Just embed the test in an inline const.